translations/tiled_cs.ts

```diff
@@ -7291,92 +7291,92 @@ Prosím vyberte formát ručně.</translation>
+        <translation>Všechny soubory musí mít stejný projektový kořen. Soubor &apos;%1&apos; nemá kořen &apos;%2&apos;.</translation>
     </message>
     <message>
         <location line="+165"/>
         <source>Cannot export tileset &apos;%1&apos; because the Godot exporter does not support collection-type tilesets.</source>
-        <translation type="unfinished"></translation>
+        <translation>Nepodařilo se exportovat sadu dlaždic &apos;%1&apos; protože exportér pro Godot nepodporuje sady obrázků.</translation>
     </message>
     <message>
         <location line="+41"/>
         <source>Godot exporter does not support property type of &apos;%1&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportér pro Godot nepodporuje typ vlastnosti &apos;%1&apos;</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Inconsistent type for property &apos;%1&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Nekonzistentní typ vlastnosti &apos;%1&apos;</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>Only objects with the resPath property will be exported</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportovány budou pouze objekty s vlastností resPath</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>resPath must be in the form of &apos;res://&lt;filename&gt;.tscn&apos;.</source>
-        <translation type="unfinished"></translation>
+        <translation>resPath musí být ve tvaru &apos;res://&lt;jméno_souboru&gt;.tscn&apos;.</translation>
     </message>
     <message>
         <location line="+58"/>
         <source>The Godot exporter does not yet support image layers</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportér pro Godot zatím nepodporuje obrázkové vrstvy</translation>
     </message>
     <message>
         <location line="+81"/>
         <source>Godot exporter only supports collisions that are rectangles or polygons.</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportér pro Godot podoporuje pro kolize jen obdélníky a mnohoúhelníky.</translation>
     </message>
     <message>
         <location line="+90"/>
         <source>Godot exporter does not support property of type &apos;%1&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Exportér pro Godot nepodporuje vlastnost typu &apos;%1&apos;</translation>
     </message>
     <message>
         <location line="+97"/>
         <source>exportAlternates is deprecated. Godot 4.2 supports native tile rotation. Tileset: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>exportAlternates už není potřeba. Godot 4.2 podporuje otáčení dlaždic přímo. Sada dlaždic: %1</translation>
```

Comment:
I translated “deprecated” as “not necessary any more”. Should it be something like “will stop working soon” instead?

Reply:
I think that translation is fine. Deprecated does imply it might be removed in the future usually, but it doesn't specify if or when.